client/state/login/actions.js

```diff
@@ -93,14 +95,22 @@ const errorFields = {
 	invalid_username: 'usernameOrEmail',
 };
 
+function getLocalizedLoginURL( action ) {
+	if ( 'en' === i18nUtils.getLocaleSlug() ) {
+		return 'https://wordpress.com/wp-login.php?action=' + action;
+	}
+
+	return 'https://' + i18nUtils.getLocaleSlug() + '.wordpress.com/wp-login.php?action=' + action;
+}
```

Comment:
We can simplify this function a bit by taking advantage of the [`addLocaleToWpcomUrl()` helper](https://github.com/Automattic/wp-calypso/blob/c5980f62c134411981506cfedc642b6c0fab7e53/client/lib/i18n-utils/utils.js#L72-L78).